From 57194f2f0663f7be6f068c2348437446b7abc19f Mon Sep 17 00:00:00 2001
From: Jernej Skrabec <jernej.skrabec@siol.net>
Date: Sun, 10 Jan 2021 12:08:58 +0100
Subject: [PATCH] configs: Add Beelink GS1 defconfig

Beelink GS1 is TV box based on H6. It includes AXP805 PMIC but it has no
power button. It comes with remote, so it's power button scancode and
protocol are adjusted.

Signed-off-by: Jernej Skrabec <jernej.skrabec@siol.net>
---
 configs/beelink_gs1_defconfig | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 configs/beelink_gs1_defconfig

diff --git a/configs/beelink_gs1_defconfig b/configs/beelink_gs1_defconfig
new file mode 100644
index 000000000000..b95d6701de9b
--- /dev/null
+++ b/configs/beelink_gs1_defconfig
@@ -0,0 +1,5 @@
+CONFIG_PLATFORM_H6=y
+CONFIG_CIR=y
+CONFIG_CIR_PROTO_NEC=y
+CONFIG_CIR_WAKE_CODE=0x8051
+CONFIG_MFD_AXP20X=y
-- 
2.30.0

